src/Symfony/Component/Form/Form.php

```diff
@@ -962,7 +962,10 @@ public function offsetUnset(mixed $name): void
      */
     public function getIterator(): \Traversable
     {
-        return $this->children;
+        // We cast to string every key to avoid issue with keys like `'0'`, `'1'`, ...
+        foreach ($this->children as $key => $value) {
```

Comment:
Could it lead to a BC break given than the returned iterator is not rewindable anymore?

Reply:
I would believe that the BC-policy is only about the typehint which is `\Traversable`.

But do you see another way to type-cast the keys to string and still keep the rewindable feature ?